pkg/cmd/infra/router/template.go

```diff
@@ -287,6 +293,99 @@ func parseCaptureHeaders(in string) ([]templateplugin.CaptureHTTPHeader, error)
 	return captureHeaders, nil
 }
 
+func parseHeadersToBeSetOrDeleted(in string) ([]templateplugin.HTTPHeader, error) {
+	var captureHeaders []templateplugin.HTTPHeader
+	var capture templateplugin.HTTPHeader
+	var err error
+	if len(in) == 0 {
+		return captureHeaders, fmt.Errorf("encoded header string not present.")
```

Comment:
This is the only error message with a period at the end.  Please remove.
```suggestion
		return captureHeaders, fmt.Errorf("encoded header string not present")
```